vault_dump/main.py

```diff
@@ -200,5 +201,20 @@ def get_audit_backends(config_root, vault_token, vault_addr):
             f.write(yaml.safe_dump(audit_details))
 
 
+def get_pki_certs(config_root, vault_token, vault_addr, mount_path):
+    # This will dump out a list of certificate IDs for a given PKI backend
+    pki_certs_response = make_request(vault_token, vault_addr, f"v1/{mount_path}/certs", "LIST")
+    if pki_certs_response.status_code in [403, 404]:
+        print('Error for getting certs from', mount_path)
+        return
+    response = pki_certs_response.json()
+    certs = response.get('data', {}).get('keys', [])
+    if certs:
```

Comment:
Missing test coverage for the case where certs exist (fixing the copy/paste in the tests will probably address this)